tsconfig.json

```diff
@@ -13,9 +13,8 @@
       "dom",
       "es6"
     ],
-    "types": [
-      "jasmine",
-      "node"
+    "typeRoots": [
```

Comment:
It allows tsc to find types for `fontfaceobserver`. I could be deliberate and add `"fontfaceobserver"` to the list, if that's preferred.